instrumentation/kafka-clients/src/main/java/brave/kafka/clients/TracingProducer.java

```diff
@@ -69,7 +69,7 @@
       if (record.key() instanceof String && !"".equals(record.key())) {
         span.tag(KafkaTags.KAFKA_KEY_TAG, record.key().toString());
       }
-      span.tag(KafkaTags.KAFKA_TOPIC_TAG, record.topic()).kind(Kind.PRODUCER).start();
+      span.tag(KafkaTags.KAFKA_TOPIC_TAG, record.topic()).name("send").kind(Kind.PRODUCER).start();
```

Comment:
these names are simply based on the java method, which is the safest choice imho